automation/common/deploy-kubevirt-and-cdi.sh

```diff
@@ -10,11 +10,10 @@ metadata:
   name: ${NAMESPACE}
 EOF
 
-# Deploying kuebvirt
-LATEST_KUBEVIRT=$(curl -L https://api.github.com/repos/kubevirt/kubevirt/releases | \
-            jq '.[] | select(.prerelease==false) | .name' | sort -V | tail -n1 | tr -d '"')
+# The version is fixed for release branch, because newer Kubevirt versions may be incompatible
+KUBEVIRT_VERSION="v0.53.2"
```

Comment:
would be possible to modify the script and get only latest 0.53 release?

Reply:
yes, good idea

Reply:
done